### ISSUE TITLE
Difference between hyp.finetune.yaml and hyp.scratch.yaml

### ISSUE THREAD

**Issue by DP1701** [ultralytics/yolov5#6820]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hello, in which case should the hyperparamterset hy.voc.yaml be used? Previously there was the hyp.finetune.yaml file, which has now disappeared. 

Namely, I have a dataset (12898 images, Train:0.65 Val: 0.2 Test 0.15) and trained two models:

- YOLOv5m6, 300 epochs, 1280px, yolov5m6.pt, hyp.scratch.yaml
- YOLOv5m6, 300 epochs, 1280px, yolov5m6.pt, hyp.finetune.yaml

I have inserted the loss graphs. I see a degradation when I take the scratch.yaml file. Are there perhaps explainable reasons why this is so? I can also provide more information if needed. 

![loss-graphs](https://user-images.githubusercontent.com/68444979/156311042-f8d73101-cada-4f7a-b53c-2ffe72b4e9ba.png)

**Comment by glenn-jocher** [comment]
@DP1701 hyp.finetune.yaml has been renamed to hyp.VOC.yaml recently. It contains hyps which are evolved specifically for finetuning (COCO pretrained) YOLOv5 models on the VOC dataset. You can see the exact command they are optimized for in the hyp yaml description:

https://github.com/ultralytics/yolov5/blob/63ddb6f0d06f6309aa42bababd08c859197a27af/data/hyps/hyp.VOC.yaml#L1-L11

TLDR: hyp.VOC.yaml is for training on VOC 50 epochs using pretrained models. hyp.scratch.yaml is for training COCO from scratch for 300 epochs.

EDIT: to use these hyps and get the best VOC training results (smaller batch sizes are also ok, i.e. 64, 32, 16):
```bash
python train.py --batch 128 --weights yolov5m6.pt --data VOC.yaml --epochs 50 --img 512 --hyp hyp.VOC.yaml
```
